src/utils.js

```diff
@@ -143,7 +140,7 @@ function loadNsecureCache(defaultPayload = {}) {
         return JSON.parse(buf.toString());
     }
 
-    const payload = Object.assign({}, cloneDeep(defaultPayload), {
+    const payload = Object.assign({}, JSON.parse(JSON.stringify(defaultPayload)), {
```

Comment:
Shouldn't do that. JSON.stringify comes with unexpected behaviors like below and is really slow.

```js
JSON.parse(JSON.stringify({ foo: Infinity}))
// {  key: null }
```

Reply:
Oh I didn't know that one :+1:  I only known about key order instability.  

About the performances you are right, `cloneDeep` is slightly more performant on small object but when the object get bigger JSON.parse can be more effective (see [this explanation](https://www.youtube.com/watch?v=ff4fgQxPaO0):
```
clone deep x 329,985 ops/sec ±2.83% (81 runs sampled)
json x 217,681 ops/sec ±2.93% (76 runs sampled)
clone deep (big object) x 58,546 ops/sec ±2.29% (79 runs sampled)
json (big object) x 56,364 ops/sec ±2.29% (83 runs sampled)
```